examples/jsm/nodes/accessors/LineMaterialNode.js

```diff
@@ -4,12 +4,6 @@ import { nodeImmutable } from '../shadernode/ShaderNode.js';
 
 class LineMaterialNode extends MaterialNode {
 
-	constructor( scope ) {
-
-		super( scope );
-
-	}
-
 	construct( builder ) {
```

Comment:
Hmm... Would it make sense to rename `construct()` to `build()`?

Reply:
I think `build` is already used for the last step -- turning the node into a shader code fragment. The `construct` name for the node constructing step seems nice 👍

Reply:
Ah, maybe `setup()` then? Or `configure()`?
Just trying to differentiate it from `constructor()`...

Reply:
`setup()` is interesting.